app/src/test/kotlin/com/flxrs/dankchat/utils/DateTimeUtilsTest.kt

```diff
@@ -0,0 +1,29 @@
+package com.flxrs.dankchat.utils
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+
+internal class DateTimeUtilsTest {
+
+    @Test
+    fun formatSeconds() {
+        var result = DateTimeUtils.formatSeconds("10")
+        Assertions.assertEquals("10s", result)
```

Comment:
Fine for this PR but just a heads up for future tests: I will probably add the `kotlin-test` dependency and use its own assertions